README.md

```diff
@@ -57,9 +58,9 @@ Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser
 
 1. Move into the ansible directory
 2. Edit inventory.txt by adding the Managed Node IP Address to deploy the app to.
-3. Create an ansible encrypted file to store your trello secrets by running
+3. Create an ansible encrypted file to store your Mongo DB secrets by running
 ```bash
-$ ansible-vault create --vault-id my_pass@prompt trello_secrets.yml
+$ ansible-vault create --vault-id my_pass@prompt mongo_secrets.yml
```

Comment:
Nice job keeping this up to date! I have to say, I think it's fair to leave this behind as we won't be using Ansible going forward, but it's up to you - feel free to just remove it if you want to, let it get outdated, or keep it up to date as you choose

Reply:
Thanks for the tip. I'll keep it as mongo_secrets.yml